scripts/force-rebuild-and-test.mjs

```
@@ -0,0 +1,139 @@
+    return false;
+  }
+}
+
+function runCommand(command, args, options = {}) {
+  return new Promise((resolve) => {
+    const process = spawn(command, args, {
+      stdio: options.captureOutput ? ['pipe', 'pipe', 'pipe'] : 'inherit',
+      cwd: process.cwd
+    });
+    
+    let output = '';
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)